build-tools/packages/build-cli/src/test/init.ts

```diff
@@ -6,14 +6,15 @@
 import { test as oclifTest } from "@oclif/test";
 
 /**
- * Initializes the oclif command test environment. @oclif/test cannot find the path to the project in some
+ * Initializes the oclif command test environment. \@oclif/test cannot find the path to the project in some
  * circumstances, so as a workaround we configure it explicitly by passing in the URL to the test module itself.
  *
  * @param moduleUrl - The URL to the test module. In most cases you should pass the `import.meta.url` value for the test
  * module when calling this function.
  *
- * @returns A test function that can be used to tst oclif commands.
+ * @returns A test function that can be used to test oclif commands.
  */
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/explicit-function-return-type
```

Comment:
`: ReturnType<typeof oclifTest.loadConfig>` seems to do the trick and we don't have to rewrite the super cumbersome type :)